autokeras/pretrained/object_detector.py

```diff
@@ -517,12 +517,8 @@ def _build_ssd(self, phase, size=300, num_classes=21):
         return SSD(phase, size, base_, extras_, head_, num_classes, self.device)
 
     def load(self, model_path=None):
-        # https://s3.amazonaws.com/amdegroot-models/ssd300_mAP_77.43_v2.pth
         if model_path is None:
-            file_link = Constant.PRE_TRAIN_DETECTION_FILE_LINK
-            # model_path = os.path.join(temp_path_generator(), "object_detection_pretrained.pth")
-            model_path = temp_path_generator() + '_object_detection_pretrained.pth'
-            download_file(file_link, model_path)
+            model_path = download_model(Constant.PRE_TRAIN_DETECTION_FILE_LINK, 'object_detection_pretrained.pth')
```

Comment:
Change this line to use a dictionary like the FACE_DETECTOR.
Constant.OBJECT_DETECTOR['']